mcux/mcux-sdk/drivers/rgpio/fsl_rgpio.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2016, Freescale Semiconductor, Inc.
- * Copyright 2016-2017, 2020 NXP
+ * Copyright 2016-2017, 2020-2021 NXP
```

Comment:
These changes to the SDK driver should also be sent to the SDK github repo. Below is the link:
https://github.com/nxp-mcuxpresso/mcux-sdk/

Reply:
@CkovMk , can you please confirm that these changes to the SDK driver have been sent to the SDK Github repo? Also please update the `README` inside `mcux` folder.

Reply:
@mmahadevan108 , not yet. I will port it here when sdk repo is updated.

Reply:
@CkovMk, we do not want to maintain these patches inside Zephyr HAL. Hence the suggestion of submitting the patch SDK github so it can be integrated there making it easier for us to update SDK code.